convert_checkpoints.py

```diff
@@ -403,16 +404,70 @@ def merge_weights(
   print(f"Export outputs takes {end - start} seconds")
 
 
+def convert_hf_gemma_weights(
```

Comment:
To be consistent with llama model, can we also convert gemma weights as safetensors?

Reply:
Code can also be re-used from convert and load if we use safe tensors format.

Reply:
For `google/gemma-7b-pytorch` it contains `torch.complex64` dtype in the weight, which is not supported by safetensors. We have to convert to pytorch state dict format for loading.

Reply:
I'll reuse the existing function to save weights in either safetensor or state_dict format, and add assertion to disallow safetensor output format for gemma model for now.

Reply:
Can you share which items are torch.complex64? If it's freqs_cis, we can ignore it and get it from pytorch model.

Reply:
Yes, it's `freqs_cis`. Let's do not include the weight in safetensor, and reconstruct while loading